src/components/SaleCard/SaleCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentProps, forwardRef, ElementType } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { Text, TextVariants } from '../Text';
+import { Link, LinkProps } from '../Link';
+import Button from '../Button/Button';
+import { ButtonVariants } from '../Button/types';
+import { SSRMediaQuery } from '../../providers/SeldonProvider/utils';
+
+export type SaleCardProps = ComponentProps<'div'> & {
```

Comment:
this should be an interface and can we put jsdoc comments describing the props? These become the docs in the storybook.